### PR TITLE
Create 1907-04-29 

### PR DESCRIPTION
**Pull request by mgh19i**
https://github.com/dig-eg-gaz/content/commit/12f0f31b9bbbc6b5866f8a92a72a3c7a9676461f

### CODE REVIEW

**Comment by whanley** [comment]
Thanks Moriah. This pull request contains three files, but should only contain one. Delete  `1907-04-29 - Shortcut.lnk` and `0 New Bitmap Image.bmp ` from your fork of the content repository, leaving only ` 1907-04-29.xml`. Then send a new pull request.

**Comment by mgh19i** [comment]
https://github.com/dig-eg-gaz/content/pull/781#

**Comment by whanley** [comment]
Thanks for your work thus far
- at line 1399, you close page 2's div as soon as you open it. Move the `</div>` tag to the end of the page, i.e. just behind the last material from the page.
- looking forward to seeing the results of your OCR on page 3. Let me know if you need help.

**Comment by whanley** [comment]
Moriah, you've done a great job getting all of the material for your first two issues into these files. I see that you've also done good work updating the templates.

There are some structural problems with these files, however. Both of them give me a red square in Oxygen, meaning that they are not well-formed.

There are a few problems. You do not close all the divs you open, so the pages are not containing all the divs they should contain. You do not use paragraphs (`<p>`) in a lot of the material that you OCRed. You also have some issues with your column breaks. Every `<cb n=""/>` needs to have a slash before the closing >. For more information on all of this, read this tutorial: https://dig-eg-gaz.github.io/how-to/digitization/polished-tei-instructions/

If you need help tidying up these files after you've gone over these instructions, check with Huaqing or me.

**Comment by whanley** [comment]
One more point: the dates in your header are not formatted correctly. I've made notes inline in your files.

File: 1907-04-29.xml [hunk 1]
```diff
@@ -0,0 +1,7098 @@
...
+        <fileDesc>
+            <titleStmt>
+                <title level="m" type="main">Digital Egyptian Gazette</title>
+                <title level="m" type="sub">An encoded transcription</title>
+                <editor>Moriah House</editor>
+                <principal>Will Hanley</principal>
+            </titleStmt>
+            <editionStmt>
+                <edition>
+                    <date>2019-09-06>September 06, 2019</date>
```
Comment (whanley) [inline 1907-04-29.xml]: <date when="2019-09-06">September 6, 2019</date>

File: 1907-04-29.xml [hunk 2]
```diff
@@ -0,0 +1,7098 @@
...
+                </edition>
+            </editionStmt>
+            <publicationStmt>
+                <publisher>FSU University Libraries</publisher>
+                <pubPlace>Tallahassee, FL</pubPlace>
+            </publicationStmt>
+            <sourceDesc>
+                <bibl>
+                    <title>The Egyptian Gazette</title>
+                    <date>1907-04-29><name>Monday</name>, April 29,
```
Comment (whanley) [inline 1907-04-29.xml]: <date when="1907-04-29">

**Comment by huoyanzhizhou** [comment]
Any recent news?

**Pull request closed without merging**